zerver/lib/bugdown/__init__.py

```diff
@@ -1526,6 +1526,12 @@ def extendMarkdown(self, md: markdown.Markdown, md_globals: Dict[str, Any]) -> N
             BacktickPattern(r'(?:(?<!\\)((?:\\{2})+)(?=`+)|(?<!\\)(`+)(.+?)(?<!`)\3(?!`))'),
             "_begin")
 
+        md.inlinePatterns.add(
+            'strong_em',
+            markdown.inlinepatterns.DoubleTagPattern(
+                r'(\*\*\*)(?!\s+)([^\*^\n]+)(?<!\s)\*\*\*', 'strong,em'),
+            '>backtick')
```

Comment:
Why is this `DoubleTagPattern` rather than the `SingleTagPattern` we have for `strong` below?

Reply:
Ahh, OK, that makes sense.